wandb/util.py

```diff
@@ -1552,6 +1553,63 @@ def load_as_json_file_or_load_dict_as_json(config: str) -> Any:
             return None
 
 
+def _parse_entity_project_item(path: str) -> tuple:
+    """Parses paths with the following formats: {item}, {project}/{item}, & {entity}/{project}/{item}.
+
+    Args:
+        path: `str`, input path; must be between 0 and 3 in length.
+
+    Returns:
+        tuple of length 3 - (item, project, entity)
+
+    Example:
+        alias, project, entity = _parse_entity_project_item("myproj/mymodel:best")
+
+        assert entity   == ""
+        assert project  == "myproj"
+        assert alias    == "mymodel:best"
+
+    """
+    words = path.split("/")
+    if len(words) == 0 or len(words) > 3:
```

Comment:
what would be a case when `len(words) == 0`? I think if it is an empty string it will be at least length 1?
maybe good idea to check that the path is of type `str`?

Reply:
ah nice -- let me remove the `len(words) == 0` check.